### PR TITLE
Rephrase the CSM Wizard steps

### PR DESCRIPTION
**Pull request by coulof**
# Description
Simplify the steps, rephrase & normalize formatting.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| |

# Checklist:

- [ ] Have you run a grammar and spell checks against your submission?
- [x] Have you tested the changes locally?
- [x] Have you tested whether the hyperlinks are working properly?
- [ ] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

File: content/docs/deployment/csminstallationwizard/_index.md [hunk 1]
```diff
@@ -65,19 +65,17 @@ The [Dell Container Storage Modules Installation Wizard](./src/index.html) is a
     helm repo update
     ```
 
-2. Copy the downloaded values.yaml file.
+2. Copy the downloaded `values.yaml` file.
 
 3. Look over all the fields in the generated `values.yaml` and fill in/adjust any as needed.
 
-4. For the Observability module, please refer [Observability](../../observability/deployment/#post-installation-dependencies) to install the post installation dependencies.
+>NOTE: The CSM Installation Wizard generates `values.yaml` with the minimal inputs required to install the CSM. To configure additional parameters in values.yaml, please follow the steps outlined in [PowerStore](../../csidriver/installation/helm/powerstore/#install-the-driver), [PowerMax](../../csidriver/installation/helm/powermax/#install-the-driver), [PowerScale](../../csidriver/installation/helm/isilon/#install-the-driver), [PowerFlex](../../csidriver/installation/helm/powerflex/#install-the-driver), [Unity XT](../../csidriver/installation/helm/unity/#install-csi-driver), [Observability](../../observability/), [Replication](../../replication/), [Resiliency](../../resiliency/).
```
Comment (sharont58) [inline content/docs/deployment/csminstallationwizard/_index.md:72]: Remove "please"

File: content/docs/deployment/csminstallationwizard/_index.md [hunk 2]
```diff
@@ -92,21 +90,19 @@ The [Dell Container Storage Modules Installation Wizard](./src/index.html) is a
...
 
->NOTE: Only the Authorization sidecar is enabled by the CSM Installation Wizard. The Proxy Server has to be installed and configured separately.
+3. If Observability is checked in the wizard, only the sidecar is enabled. Please refer to [Observability](../../observability/deployment/#post-installation-dependencies) to scrape metrics to Prometheus and load the Grafana dashboards.
```
Comment (sharont58) [inline content/docs/deployment/csminstallationwizard/_index.md:101]: ...to **export** metrics ...

**Review by sharont58 — DISMISSED** [review]
Global comment: In technical documentation, we do not use the term "please". Instead, just tell the reader what to do. 
For example, line 74:

If Observability is checked in the wizard, only the sidecar is enabled. Refer to [Observability](../../observability/deployment/#post-installation-dependencies) to **export** metrics to Prometheus and load the Grafana dashboards.

File: content/docs/deployment/csminstallationwizard/_index.md (hunk 2 above)
Comment (coulof) [reply content/docs/deployment/csminstallationwizard/_index.md]: Done

**Comment by coulof** [comment]
> Global comment: In technical documentation, we do not use the term "please". Instead, just tell the reader what to do. For example, line 74:
> 
> If Observability is checked in the wizard, only the sidecar is enabled. Refer to [Observability](../../observability/deployment/#post-installation-dependencies) to **export** metrics to Prometheus and load the Grafana dashboards.

Thanks for the comment, this is now fixed.

**Review by gallacher — DISMISSED** [review]
LGTM

**Review by donatwork — DISMISSED** [review]

**Comment by shanmydell** [comment]
@coulof : Please resolve merge conflicts

**Comment by coulof** [comment]
> @coulof : Please resolve merge conflicts

Done @shanmydell  & @gallacher

**Review by gallacher — APPROVED** [review]

**Review by shaynafinocchiaro — APPROVED** [review]

**Review by shanmydell — APPROVED** [review]

File: content/docs/deployment/csminstallationwizard/_index.md [hunk 3]
```diff
@@ -100,19 +100,19 @@ The [Dell Container Storage Modules Installation Wizard](./src/index.html) is a
...
 
-4. If Authorization is enabled, refer to [Authorization](../csmoperator/modules/authorization) for installation and configuration of the Proxy Server.
+3. If Observability is checked in the wizard, only the sidecar is enabled. Refer to [Observability](../../observability/deployment/#post-installation-dependencies) to export metrics to Prometheus and load the Grafana dashboards.
 
->NOTE: Only the Authorization sidecar is enabled by the CSM Installation Wizard. The Proxy Server has to be installed and configured separately.
+4. If Authorization is checked in the wizard, only the sidecar is enabled. Refer to [Authorization](../../authorization/deployment/helm/) to install and configure the CSM Authorization Proxy Server.
 
-5. If Replication is enabled, refer to [Replication](../csmoperator/modules/replication) for the necessary prerequisites required for this module.
+5. If Replication is checked in the wizard, only the sidecar is enabled. Refer to [Replication](../../replication/deployment/) on how to configure communication between Kubernetes clusters and the replication controller.
```
Comment (santhoshatdell) [inline content/docs/deployment/csminstallationwizard/_index.md:115]: Same as above. Module gets installed too. Please check the behavior of other modules.
I also see the change in reference link. It should point to the module info from CSM Operator like before.

**Pull request merged**